logback-classic/src/main/java/ch/qos/logback/classic/net/SyslogAppender.java

```diff
@@ -37,15 +37,20 @@
 
   static final public String DEFAULT_SUFFIX_PATTERN = "[%thread] %logger %msg";
   static final public String DEFAULT_STACKTRACE_PATTERN = "" + CoreConstants.TAB;
+  static final public String DEFAULT_STACKTRACE_HEADER_PATTERN = "";
 
+  PatternLayout stackTraceHeaderLayout = new PatternLayout();
+  String stackTraceHeaderPattern = DEFAULT_STACKTRACE_HEADER_PATTERN;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/tony19/logback/file/2376021715/issues/source?bid=3222618&fileBranchId=3222713#l43)